app/[locale]/academics/departments/[name]/page.tsx

```diff
@@ -35,14 +27,23 @@ export default async function Department({
   const department = await db.query.departments.findFirst({
     where: (departments, { eq }) => eq(departments.urlName, name),
   });
+  const departmentHod = await db.query.hod.findFirst({
+    where: (hod, { eq }) => eq(hod.departmentId, department?.id ?? 0),
+    with: {
+      faculty: {
+        columns: {
+          id: true,
+          officeAddress: true,
+          officeTelephone: true,
+          outreach: true,
+        },
+      },
+    },
+  });
   // if (!department) notFound(); // FIXME: Remove this once dynamicParams works
```

Comment:
Uncomment this line and remove the `?` after the department variable wherever used.

Reply:
sir I tried both the squash and drop method and they does change in my local branch, but at the same time git create that many pull requests from the remote origin. how to get rid of this?

Reply:
> but at the same time git create that many pull requests from the remote origin

What does this mean?